nmdc_server/fakes.py

```diff
@@ -222,25 +222,25 @@ class Meta:
         model = models.MetagenomeAssembly
         sqlalchemy_session = db
 
-    scaffolds: int = Faker("pyint")
-    contigs: int = Faker("pyint")
-    scaf_bp: int = Faker("pyint")
-    contig_bp: int = Faker("pyint")
-    scaf_n50: int = Faker("pyint")
-    scaf_l50: int = Faker("pyint")
-    ctg_n50: int = Faker("pyint")
-    ctg_l50: int = Faker("pyint")
-    scaf_n90: int = Faker("pyint")
-    scaf_l90: int = Faker("pyint")
-    ctg_n90: int = Faker("pyint")
-    ctg_l90: int = Faker("pyint")
-    scaf_max: int = Faker("pyint")
-    ctg_max: int = Faker("pyint")
-    scaf_n_gt50k: int = Faker("pyint")
-    scaf_l_gt50k: int = Faker("pyint")
-    scaf_pct_gt50k: int = Faker("pyint")
-    num_input_reads: int = Faker("pyint")
-    num_aligned_reads: int = Faker("pyint")
+    scaffolds: int = Faker("pyfloat")
+    contigs: int = Faker("pyfloat")
+    scaf_bp: int = Faker("pyfloat")
+    contig_bp: int = Faker("pyfloat")
+    scaf_n50: int = Faker("pyfloat")
+    scaf_l50: int = Faker("pyfloat")
+    ctg_n50: int = Faker("pyfloat")
+    ctg_l50: int = Faker("pyfloat")
+    scaf_n90: int = Faker("pyfloat")
+    scaf_l90: int = Faker("pyfloat")
+    ctg_n90: int = Faker("pyfloat")
+    ctg_l90: int = Faker("pyfloat")
+    scaf_max: int = Faker("pyfloat")
+    ctg_max: int = Faker("pyfloat")
+    scaf_n_gt50k: int = Faker("pyfloat")
+    scaf_l_gt50k: int = Faker("pyfloat")
+    scaf_pct_gt50k: int = Faker("pyfloat")
+    num_input_reads: int = Faker("pyfloat")
+    num_aligned_reads: int = Faker("pyfloat")
```

Comment:
@naglepuff should these be type `float` instead of `int` now?

Reply:
Good catch - these should all be correctly typed as of a816215